src/fragments/ui/storage/web/s3-image.mdx

```diff
@@ -32,6 +32,11 @@ _app.component.html_
 ```html
 <amplify-s3-image img-key="example.png"></amplify-s3-image>
 ```
+_app.component.html_ (using data binding e.g. filename)
+```html
+<amplify-s3-image imgKey="{{filename}}"></amplify-s3-image>
```

Comment:
Looks good, though I think data binding makes is more idiomatic here than interpolation:

```suggestion
<amplify-s3-image [imgKey]="filename"></amplify-s3-image>
```